onnxruntime/test/providers/cpu/tensor/space_depth_ops_test.cc

```diff
@@ -35,7 +35,8 @@ TEST(TensorOpTest, SpaceToDepthTest_1) {
       1.1f, 1.3f,
       3.1f, 3.3f};
   test.AddOutput<float>("output", {N, C * blocksize * blocksize, H / blocksize, W / blocksize}, result);
-  test.Run();
+  // TensorRT has error: Expected output shape [{1,8,1,2}] did not match run output shape [{8,1,1,2}] for output
```

Comment:
is this a regression or is this new TRT7 support which doesn't work?

Reply:
The issue has been reported to Nvidia. It's like a new issue.